### ISSUE TITLE
Consider adding variations in theme screenshot

### ISSUE THREAD

**Issue by beafialho** [WordPress/twentytwentythree#163]
It could be nice to make the screenshot a little more elucidative so that when a user installs it, they understand they have all those options to choose from. I tried searching for screenshot requirements and found [this](https://codex.wordpress.org/Theme_Development#Screenshot).

Even though it says to keep the graphics near the center, I thought these were a nice way to present them.

![TT3 Screenshot-1](https://user-images.githubusercontent.com/65220155/189876618-74c7504a-bc28-4318-83fe-f47f128a9140.png)
![TT3 Screenshot-2](https://user-images.githubusercontent.com/65220155/189876631-dcaedd30-5eca-4214-9751-e0caafd4f979.png)

Here are some not so obvious versions:

![TT3 Screenshot-3](https://user-images.githubusercontent.com/65220155/189876634-dcc73339-bf37-4602-a299-022330ca58dc.png)
![TT3 Screenshot-4](https://user-images.githubusercontent.com/65220155/189877693-51c683a5-767e-4f9e-ac39-30a6611a4498.png)

**Comment by carolinan** [comment]
The theme review requirements are not listed on the old codex page, but here:
https://make.wordpress.org/themes/handbook/review/required/

```
Screenshot

The screenshot must not look like an advertisement. The reviewer can subjectively ask you to change screenshots if they find that it is not appropriate.
The screenshot must not be bigger than 1200 x 900px
The ratio of width to height needs to be 4:3
```

**Comment by carolinan** [comment]
Generally, the theme screenshot should look like the theme when it is installed, OR, of a design that is possible to create with the theme options. It should not be a dummy image.

Please also consider that default themes sets precedences, this may have an unwanted result, that themes will use screenshots that do not look like the actual theme. - This requirement is in place because people used screenshots that were misleading for users.

**Comment by critterverse** [comment]
>Generally, the theme screenshot should look like the theme when it is installed, OR, of a design that is possible to create with the theme options. It should not be a dummy image.

That's good to know, thanks @carolinan. Since a lot of new users are going to get TT3 with a fresh install, I think it would be helpful to figure out some way to show all the variations within the screenshot as a way of helping educate folks that the base theme is meant to be a "blank canvas" for the style variations. I wonder if something more like a literal screenshot would work but split into slivers that show a peek of the different styles available (but combined to make a singular image).

In general, this seems like this could be a good approach for other theme authors too (at least until we can figure out how to showcase style variations in the theme directory somehow). That way, the restriction of it needing to look like a real layout from the theme could remain the guiding principle but folks could get more creative with how they create that image.

**Comment by beafialho** [comment]
> Please also consider that default themes sets precedences, this may have an unwanted result, that themes will use screenshots that do not look like the actual theme. - This requirement is in place because people used screenshots that were misleading for users.

Those are great points @carolinan and @critterverse, maybe we can have a literal screenshot then:

![TT3 Screenshot](https://user-images.githubusercontent.com/65220155/189925040-e5dc1cec-aa89-4f8b-be4d-4a6644a93558.png)

**Comment by critterverse** [comment]
^ Love that idea!

**Comment by carolinan** [comment]
That could work.
Related: https://meta.trac.wordpress.org/ticket/6059

**Comment by tinjure20** [comment]
Just a suggestion. May the above @beafialho suggested images could be added as post featured images. IMO, these images too beautiful not to include in the screenshot. I'm not sure, that violates theme requirement for screenshot (cc:  @carolinan).

**Comment by tinjure20** [comment]
Screenshot sample 1:

<img width="600" alt="Screen Shot" src="https://user-images.githubusercontent.com/59660253/190032476-eed6ba4f-53a7-4de0-a8b3-fda25ddbf2c8.png">

sample 2:

<img width="1221" alt="Screenshot-2" src="https://user-images.githubusercontent.com/59660253/190032558-572a99a0-b466-4801-b9e3-42cec1d6cb41.png">

**Comment by beafialho** [comment]
I understand the idea but I think it can easily become redundant and confusing. My suggestion is to just use the screenshot of the theme and the variations in the sidebar.

**Comment by beafialho** [comment]
Closing this as it has been addressed in #176
